### PR TITLE
mo' tguis: electric boogaloo

### PR DESCRIPTION
**Pull request by neersighted**
Also re-balances cryo/freezer specific heat to more sane levels (yay freezer over).

### CODE REVIEW

**Comment by WJohn** [comment]
At this point I don't think I can even pretend to understand what you're doing.

That's okay, pretty pictures.

**Comment by neersighted** [comment]
@WJohn the color is based on 'status', ie good/average/bad, and blue is average. The colors are per-theme so there's really no way to say 'blue' because one theme's average blue is another theme's average yellow.

**Comment by PKPenguin321** [comment]
Do the little things in all the department head offices that you have to swipe for red alert have a new UI yet? I'd argue that they need it the most.

**Comment by duncathan** [comment]
@PKPenguin321 #14824

File: code/modules/tgui/states/physical.dm [hunk 1]
```diff
@@ -19,3 +19,6 @@
 
 /mob/living/silicon/physical_can_use_topic(src_object)
 	return max(UI_UPDATE, shared_living_ui_distance(src_object)) // Silicons can always see.
+
+/mob/living/silicon/ai/physical_can_use_topic(src_object)
+	return UI_CLOSE // AIs are not physical.
```
Comment (GunHog) [inline code/modules/tgui/states/physical.dm]: What exactly does this mean here? What does it restrict?

Comment (neersighted) [inline code/modules/tgui/states/physical.dm]: It keeps AIs from opening any UI that uses `physical_state`, such as canisters or keycard auth devices.

Comment (PKPenguin321) [inline code/modules/tgui/states/physical.dm]: Wait, so this makes the AI unable to use fire alarms? That's not right...

Comment (neersighted) [inline code/modules/tgui/states/physical.dm]: Fire alarms are not physical, they're `default_state`.

Comment (GunHog) [inline code/modules/tgui/states/physical.dm]: That is disappointing. Canisters have an electronic interface with physical controls, and AIs could never *use* the canisters, but could still read them. Oh well :(

Comment (neersighted) [inline code/modules/tgui/states/physical.dm]: Fair enough, I'll let AIs have UI_UPDATE so they can look at it.

Comment (GunHog) [inline code/modules/tgui/states/physical.dm]: <3

**Pull request merged**